app/controllers/concerns/translatable.rb

```diff
@@ -2,7 +2,6 @@ module Translatable
   extend ActiveSupport::Concern
 
   included do
-    before_action :set_translation_locale
```

Comment:
What will happen if we don't delete this callback? (as well as its related action)

Reply:
With these changes, this callback would be redundant, since the same thing is now done in `ApplicationController`.

Reply:
Before these changes, `ApplicationController#set_locale` configured `I18n.locale` and `Translatable#set_translation_locale` configured `Globalize.locale`.

After these changes, `ApplicationController#set_locale` configures both `I18n.locale` and `Globalize.locale`. So there's no need for `Translatable#set_translation_locale` anymore, since `Globalize.locale` is already configured.